inmemory/src/main/java/org/vertexium/inmemory/InMemoryVertex.java

```diff
@@ -54,8 +50,7 @@ public InMemoryVertex(
     @Override
     public Iterable<EdgeInfo> getEdgeInfos(Direction direction, final String[] labels, Authorizations authorizations) {
         if (!getFetchHints().isIncludeEdgeRefs()) {
```

Comment:
Should this be using `getFetchHints().validateHasEdgeFetchHints(direction)` instead? If so, see above about checking the labels too.

Reply:
done